README.md

```diff
@@ -1 +1,52 @@
-# Module1Capstone
\ No newline at end of file
+# Capstone Project
+
+> In this capstone project you are going to build is based on an online website for a conference. We provide some design guidelines for you to create the website, but you must personalize the content, i.e., instead of a conference you can build a website for a concert or for a web development course. It's very important the you personalize your project so that you have something unique in your portfolio to share with potential employers during job searching.
+
```

Comment:
**OPTIONAL**
- It'd be very nice if you added a couple of images of the app under this nice description 😉